trackintel/io/postgis.py

```diff
@@ -50,7 +50,7 @@ def read_positionfixes_postgis(
     parse_dates=None,
     params=None,
     chunksize=None,
-    **kwargs
+    read_gpd_kws=None,
```

Comment:
I think it would be more clean if we set the defaul to `{}`?  Then in the code below, we could drop the `(read_gpd_kws or {})` and just put `read_gpd_kws` instead.

Reply:
Mutable datatypes as default arguments is a python pitfall.
Functions are only initiated once, thus all function calls share the same object.
This can cause behavior like this
````python
>>> def foo(a=[]):
...     a.append(1)
...     print(a)
...
>>> foo()
[1]
>>> foo()
[1, 1]
````

Reply:
Interesting, didn't know this! I guess it wouldn't matter in our case since we are not mutating the dictionary anywhere in the function, but I'm fine with the current version if you think it's better code style :)

Reply:
Yes, I would keep it this way. More as an argument of code style then something else. :)